R/geom_textsf.R

```diff
@@ -141,20 +141,16 @@ GeomTextsf <- ggproto("GeomTextSf", GeomSf,
 
   required_aes = c("geometry", "label"),
 
+
+
   default_aes = aes(
-    shape      = NULL,
-    colour     = "gray30",
-    fill       = "gray90",
-    linetype   = 1,
-    stroke     = 0.5,
-    size       = 3.88,
+    !!!GeomSf$default_aes[-which(names(GeomSf$default_aes) %in% c("size", "linewidth"))],
+    !!!GeomText$default_aes[c("family", "size")],
+    linewidth  = !!GeomLine$default_aes$linewidth,
```

Comment:
The new ggplot2 version has more unevaluated expressions that take their values from the theme.
We should ideally copy defaults from ggplot2's Geoms wherever we can, but it'd be easier when the new release is out.